src/components/Blocking/SpecificIssue.js

```diff
@@ -61,13 +62,21 @@ const styles = StyleSheet.create({
     marginLeft: 18,
     marginBottom: 19
   },
+  containerBtnText: {
+    marginRight: 22,
+    marginLeft: 18,
+    marginBottom: 19,
+    fontFamily: fonts.inter[500],
+    fontSize: 16,
+    color: COLORS.white2
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `containerBtnText` style is good for maintaining consistency and readability. However, it seems like the color `COLORS.white2` might not be accessible on all backgrounds. Please ensure that this color provides enough contrast against its background for accessibility purposes.



<!-- This is an auto-generated comment by OSS CodeRabbit -->